python/sbp/client/framer.py

```diff
@@ -128,7 +128,7 @@ def _receive(self):
             pass
         return msg
 
-    def __call__(self, msg, **metadata):
+    def __call__(self, *msgs, **metadata):
```

Comment:
Please update the docstring, too.

I presume that metadata will be shared between the messages sent in one go if it ever gets implemented?

Reply:
Is there a guaranteed ordering of the messages?

Reply:
Messages will be passed to the transport in the order that they appear in the list, beyond that I don't think this changes anything with regards to ordering guarantees (that we had prior).

Reply:
@tn-exafore I was thinking we should just remove the metadata parameter... @denniszollo any comments on that?  Can I just remove `metadata`?

Reply:
Removing metadata was too much of a rats nest, seems some of this data is passed to logging objects at some point.

Reply:
(Also, fixed docs)